docs/bqetl.md

```diff
@@ -0,0 +1,45 @@
+# bqetl CLI
```

Comment:
Can you reference this in the README, e.g. via a link? 

https://github.com/mozilla/bigquery-etl#the-bqetl-cli